tests/python/frontend/tflite/test_forward.py

```diff
@@ -2652,6 +2652,77 @@ def test_forward_reverse_v2():
         _test_reverse_v2((5, 6, 4, 2), np.array([2], dtype='int32'), dtype)
 
 
+#######################################################################
+# MATRIX_SET_DIAG
+# ---------------
+
+def _test_matrix_set_diag(input_shape, input_type, quantized=False):
+    """ One iteration of MATRIX_SET_DIAG """
+    with tf.Graph().as_default():
+        diagonal_shape = list(input_shape[:-2])
+        diagonal_shape.append(min(input_shape[-2], input_shape[-1]))
```

Comment:
Should the broadcasting case be tested here?

Reply:
TFLite MATRIX_SET_DIAG doesn't seem to be a broadcast operator. So, I'll change the registration to be injective.